### PR TITLE
ISSUE_TEMPLATE: add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by xiekeyi98**
add ISSUE_TEMPLATE to make ISSUE clear.

现在的issues里，有很多和docs-cn无关的内容，很多提问TiDB、TiKV等的相关问题。

通过这个PR，希望能对仓库的issues进行一下引导。

### CODE REVIEW

**Comment by winkyao** [comment]
@xiekeyi98 Please change the language from chinese to English?

**Comment by xiekeyi98** [comment]
> @xiekeyi98 Please change the language from chinese to English?

Many people in thie repo use Chinese to discuss.
I think use Chinese is better , do you ?

**Comment by xiekeyi98** [comment]
@TomShawn  PTAL.
I was wondering if I could ask you to review my poor english describe?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+name: "\U0001F41B Bug Report"
+about: Typo, Grammatical error, etc.。
 
 ---
 
 ## Bug 报告
 
-**本仓库仅用于解决与中文文档相关的问题，其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/)**
+**This repository is ONLY used to solve problems related to DOCS-CN.
+PLease move other questions(TiDB, TiKV, etc) to [other repositories](https://github.com/pingcap/).**
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+<!-- 本仓库仅用于解决与中文文档相关的问题，
+其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/) -->
 
-非常感谢你提出相关问题，在提交你的issue之前，请回答下面几个问题:
+Please answer these questions before submitting your issue. Thanks!
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Please answer the following questions before submitting your issue. Thanks!

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+<!-- 非常感谢你提出相关问题，在提交你的issue之前，请回答下面几个问题: -->
 
-1. 该问题出现的文档路径是什么？
+1. What is the document path for this problem?
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: 1. What is the document directory for the problem?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
+<!-- 1. 该问题出现的文档路径是什么？-->
 
 
-2. 你希望它应该是怎样的?
+2. How do you want to improve?
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: 2. How would you like to improve it?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 5]
```diff
@@ -1,19 +1,25 @@
 ---
 name: "\U0001F680 Feature Request"
-about: 有一个更好的建议？希望扩充某一部分文档内容?
+about: Have a suggestion? Want to expand the content?
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/feature-request.md]: about: Any suggestion? Want to add something new?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 6]
```diff
@@ -1,19 +1,25 @@
...
 
 ---
 
 ## Feature Request
 
-**本仓库仅用于解决与中文文档相关的问题，其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/)** 
+**This repository is ONLY used to solve problems related to DOCS-CN.
+PLease move other questions(TiDB, TiKV, etc) to [other repositories](https://github.com/pingcap/).**
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/feature-request.md]: 见上

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 7]
```diff
@@ -1,19 +1,25 @@
...
+<!-- 本仓库仅用于解决与中文文档相关的问题，
+其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/) -->
 
-1. 提出的扩充文档部分，是因为你遇到了相关问题吗？如果是，请描述一下你所遇到的问题。
+1. Is your feature request related to a problem? If yes, please describe it.
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/feature-request.md]: 1. Please describe the problem that relates to your addition.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 8]
```diff
@@ -1,19 +1,25 @@
...
+<!-- 1. 提出的扩充文档部分，是因为你遇到了相关问题吗？如果是，请描述一下你所遇到的问题。-->
 
 
-2. 描述你想要扩充的方面或你的建议。
+2. Describe the suggestion or what you want to expand.
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/feature-request.md]: 2. Please describe your suggestion or addition.

File: .github/ISSUE_TEMPLATE/question.md [hunk 9]
```diff
@@ -6,12 +6,20 @@ about: 询问那些没有在文档或其他参考资料中有答案的问题
...
+其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/) -->
 
-在提出一个问题之前，请确定你已经进行了下面的操作:
-- 在Stack Overflow等地搜索了问题。
-- 通过谷歌等搜索引擎查询了问题。
-- 查阅了所有的issues。
-- 浏览了相关的文档:
+Before asking a question, make sure you have:
+<!-- 在提出一个问题之前，请确定你已经进行了下面的操作: -->
+- Searched existing Stack Overflow questions.
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/question.md]: - Searched question in the existing Stack Overflow.

File: .github/ISSUE_TEMPLATE/question.md [hunk 10]
```diff
@@ -6,12 +6,20 @@ about: 询问那些没有在文档或其他参考资料中有答案的问题
...
+<!-- - 在Stack Overflow等地搜索了问题。 -->
+- Googled your question.
+<!-- - 通过谷歌等搜索引擎查询了问题。  -->
+- Searched open and closed GitHub issues
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/question.md]: - Searched relevant GitHub issues

File: .github/ISSUE_TEMPLATE/question.md [hunk 11]
```diff
@@ -6,12 +6,20 @@ about: 询问那些没有在文档或其他参考资料中有答案的问题
...
+<!-- - 查阅了相关的issues。 -->
+- Read the documentation:
```
Comment (TomShawn) [inline .github/ISSUE_TEMPLATE/question.md]: - Read relevant documentation:

File: .github/ISSUE_TEMPLATE/question.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+<!-- 本仓库仅用于解决与中文文档相关的问题，
+其他方面问题(如TiDB、TiKV等)请移步其他[有关仓库](https://github.com/pingcap/) -->
+
+Before asking a question, make sure you have:
+<!-- 在提出一个问题之前，请确定你已经进行了下面的操作: -->
+- Searched existing Stack Overflow questions.
+<!-- - 在Stack Overflow等地搜索了问题。 -->
+- Googled your question.
+<!-- - 通过谷歌等搜索引擎查询了问题。  -->
+- Searched relevant GitHub issues.
```
[the remark this answers is not in the document]
Comment (lilin90) [reply .github/ISSUE_TEMPLATE/question.md]: The previous version is better and clearer, and is also used in tidb and tikv repos. It'd be better to keep them consistent.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+1. Please describe the problem that relates to your addition.
+<!-- 1. 提出的扩充文档部分，是因为你遇到了相关问题吗？如果是，请描述一下你所遇到的问题。-->
+
+
+2. Please describe your suggestion or addition.
+<!-- 2. 描述你想要扩充的方面或你的建议。-->
+
+
+3. If convenient, please provide reference materials(Blog, Stack Overflow, etc).
+<!-- 3. 如果方便的话，请提供一些参考资料(如你遇到的博客、搜索到的资料等)。-->
```
[the remark this answers is not in the document]
Comment (xiekeyi98) [reply .github/ISSUE_TEMPLATE/feature-request.md]: I am sorry.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F41B Bug Report"
+about: Typo, Grammatical error, etc.。
+
```
[the remark this answers is not in the document]
Comment (xiekeyi98) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I think this blank line is enough.
like this in [tidb](https://github.com/pingcap/tidb/pull/9021/files)

[the remark this answers is not in the document]
Comment (lilin90) [reply .github/ISSUE_TEMPLATE/bug-report.md]: @xiekeyi98 Does this blank line has any meaning? If not, we can remove it to keep the doc clean.

**Pull request merged**